scopus-import/src/main/java/no/sikt/nva/scopus/conversion/model/CorporationWithContributors.java

```diff
@@ -102,7 +103,7 @@ private List<Corporation> generateCorporationFromAuthorGroupTp() {
             organizationName -> Map.of(guessTheLanguageOfTheInputStringAsIso6391Code(organizationName),
                                        organizationName))
                          : extractCountryNameAsAffiliation();
-        return isNotNorway(labels.orElse(Map.of()))
+        return isNotNorway(labels.orElse(Map.of())) && name.isPresent() && !name.get().isBlank()
                    ? List.of(new UnconfirmedOrganization(name.orElse(null)))
```

Comment:
We have to handle nullpointer exception, or I misunderstood?

Reply:
Since you added the `name.isPresent()`-check, you don't need the `.orElse()` method?
```
isNotNorway(labels.orElse(Map.of())) && name.isPresent() && !name.get().isBlank()
                   ? List.of(new UnconfirmedOrganization(name.get()))
                   : List.of();
```
Also, `name.isPresent() && !name.get().isEmpty()` is repeated twice, extract to method?

Reply:
Now I get it, thank you :)